packages/server/src/async/producers/feed-items/member-joined-to-community.spec.js

```diff
@@ -0,0 +1,7 @@
+const { MemberJoinedToCommunity } = require('./member-joined-to-community');
+
+describe('Member Joined To Community Feed Item Producer', () => {
```

Comment:
Just so I understand this right, this PR will only have "Member joining communities" as the one kind of feed item? So we'll have other stories for the other feed item types?

Reply:
Yes, sure it is like a example.